Dockerfile.template

```diff
@@ -108,9 +108,8 @@ RUN sudo mkdir -p "$PHP_INI_DIR/conf.d"  && \
 	sudo chmod +x /usr/local/bin/docker-php-* /usr/local/bin/install-php-extensions
 
 # Install the PHP package manager Composer
-ENV COMPOSER_VERSION 2.1.12
 RUN sudo php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
-	sudo php composer-setup.php --version=$COMPOSER_VERSION --install-dir=/usr/local/bin --filename=composer && \
+	sudo php composer-setup.php --install-dir=/usr/local/bin --filename=composer && \
```

Comment:
could you please use the COMPOSER_VERSION variable and set it to 2.5.1 as that's the latest available version? It doesn't seem like there are any compatibility issues. we'd like to keep the environment variable and install command since the builds are intended to be deterministic

additionally, when you make the change, could you also please rebase on main so you are also pulling in the latest changes and make sure there are no merge conflicts?